src/app/_components/NewNavBar/MobileNavPage.tsx

```diff
@@ -46,9 +46,10 @@ const MobileContentTop = ({
 
         <motion.div
           key="settings-menu"
-          initial={{ opacity: 0 }}
+          initial={{ opacity: 0, zIndex: -1, position: 'relative' }}
```

Comment:
Is it possible to do this with display none and display block? Manipulating z-indices can get messy over time